### PR TITLE
[Tests] Implement postgres driver and check for mysql

### PR DESCRIPTION
**Pull request by rskuipers**
This PR implements the PgSql driver for tests to support PostgreSQL.
It also adds a check for MySql when purging the database for the Behat tests.

### CODE REVIEW

File: src/Sylius/Bundle/ResourceBundle/Behat/DefaultContext.php [hunk 1]
```diff
@@ -73,12 +73,17 @@ public function purgeDatabase(BeforeScenarioScope $scope)
         $entityManager = $this->getService('doctrine.orm.entity_manager');
         $entityManager->getConnection()->getConfiguration()->setSQLLogger(null);
 
-        $entityManager->getConnection()->executeUpdate("SET foreign_key_checks = 0;");
+        $isMysql = $entityManager->getConnection()->getDriver() instanceof \Doctrine\DBAL\Driver\PDOMySql\Driver;
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/ResourceBundle/Behat/DefaultContext.php]: Could you add an use statement?

Comment (rskuipers) [inline src/Sylius/Bundle/ResourceBundle/Behat/DefaultContext.php]: @pjedrzejewski done

**Comment by pjedrzejewski** [comment]
@rskuipers Could you rebase? Thanks! :+1:

**Comment by rskuipers** [comment]
@pjedrzejewski Done!

**Comment by pjedrzejewski** [comment]
Thank you Rick! :+1:

**Pull request merged**